ArduCopter/takeoff.cpp

```diff
@@ -36,7 +36,7 @@ bool Mode::do_user_takeoff(float takeoff_alt_cm, bool must_navigate)
     }
 
     // Helicopters should return false if MAVlink takeoff command is received while the rotor is not spinning
```

Comment:
This comment is now out-of-date.

Should probably just remove it - given it wasn't heli-specific it was *already* wrong....

Reply:
Thanks, I've fixed the comment in a follow-up commit.